Misc/mypy/README.md

```diff
@@ -11,6 +11,17 @@ like `types`, `typing`, and `collections.abc`.
 
 So instead, we set `mypy_path` to include this directory,
 which only links modules and packages we know are safe to be
-type-checked themselves and used as dependencies.
+type-checked themselves and used as dependencies.  See
+`Lib/_pyrepl/mypy.ini` for an example.
 
-See `Lib/_pyrepl/mypy.ini` for an example.
\ No newline at end of file
+At the same time, we don't want symlinks to be checked into the
+repository as they would end up being shipped as part of the source
+tarballs, which can create compatibility issues with unpacking on
+operating systems without symlink support.  Additionally, those symlinks
+would have to be part of the SBOM, which we don't want either.
+
+Instead, this directory ships with a `make_symlinks.py` script, which
```

Comment:
FWIW, I would be inclined to not symlink anything, and instead copy files to a separate directory to run mypy on... but then I don't use mypy and I have no idea how that would integrate with normal edit-typecheck workflows people have.

Reply:
copying files makes it annoying for a local workflow since then you have to copy the tree for every mypy run. that's too slow.